vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,214 @@
+    skos:broader gmps:granted,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Renewal withdrawn"@en .
+
+gmps:replacement-permit-application a skos:Concept ;
+    skos:definition "The application represents a Replacement Permit Application."@en ;
+    skos:broader gmps:application,
+        gmps:granted ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Repalcement Permit Application"@en .
+
+gmps:unfinalised a skos:Concept ;
+    skos:definition "The Resource Permit has been terminated, but there are some outstanding requirements to be completed, to be compliant with the resources legislation."@en ;
+    skos:broader gmps:non-current,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Unfinalised"@en .
+
+gmps:unsuccessful-application a skos:Concept ;
+    skos:definition "The application represents an unsuccessful application."@en ;
+    skos:broader gmps:application,
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Unsuccessful application"@en .
+
+gmps:withdrawn a skos:Concept ;
+    skos:definition "The application has been withdrawn."@en ;
+    skos:broader gmps:non-current ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Withdrawn"@en .
+
+gmps:not-accepted a skos:Concept ;
+    skos:definition "No definition added."@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Not accepted"@en ;
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:discarded a skos:Concept ;
+    skos:definition "The application has been discarded from the system."@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Discarded"@en ;
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:prelodgement a skos:Concept ;
+    skos:definition "The application has been started in MyMinesOnline, but has not yet been lodged."@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Prelodgement"@en ;
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:submitted a skos:Concept ;
+    skos:definition "An application has been started in MyMinesOnline, but has not yet been lodged."@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Submitted"@en ;
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:granted a skos:Concept ;
+    skos:definition "The Resource Permit has been approved under the Resource Authority Legislation"@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Granted"@en ;
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:application a skos:Concept ;
+    skos:definition "An application has been submitted and an outcome is pending."@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Application"@en ;
+    skos:topConceptOf gmps:conceptScheme .
+
+gmps:non-current a skos:Concept ;
+    skos:definition "The Resource Permit has been terminated in the system and is longer current."@en ;
+    skos:inScheme gmps:conceptScheme ;
+    skos:prefLabel "Non-current"@en ;
+    skos:topConceptOf gmps:conceptScheme .
```

Comment:
I think these are all collections, rather than true concepts

Reply:
They're the top concepts. The user would select one of these from the Status drop-down list.

Reply:
Ok. But on the record I'm conceptually uncomfortable with an unspecified granted or application status when the department controls the available options so we can make people select the appropriate level of specificity.